xml/templates/message_templates/event_offline_receipt_text.tpl

```diff
@@ -38,7 +38,7 @@
 ==========================================================={if !empty($pricesetFieldsCount) }===================={/if}
 
 {event.title}
-{event.start_date|crmDate}{if $event.event_end_date}-{if $event.event_end_date|crmDate:"%Y%m%d" == $event.event_start_date|crmDate:"%Y%m%d"}{$event.event_end_date|crmDate:0:1}{else}{$event.event_end_date|crmDate}{/if}{/if}
+{event.start_date|crmDate}{if {event.end_date|boolean}}-{if '{event.end_date|crmDate:"%Y%m%d"}' === '{event.start_date|crmDate:"%Y%m%d"}'}{event.end_date|crmDate:"Time"}{else}{event_end_date}{/if}{/if}
```

Comment:
At the end of this line: `{event_end_date}`

Reply:
thanks @demeritcowboy 

Re-assuring jenkins also spotted it - CRM_Event_Form_ParticipantTest::testSubmitWithDeferredRecognition
Exception: CRM_Core_Exception: "Message was not parsed due to invalid smarty syntax : Smarty error: [in evaluated template line 41]: syntax error: unrecognized tag &#039;event_end_date&#039; (Smarty_Compiler.class.php, line 584)"
#0 [internal function](): CRM_Core_Smarty->handleSmartyError(256, "Smar